helm/default-apps-cloud-director/values.yaml

```diff
@@ -156,6 +156,21 @@ apps:
     # used by renovate
     # repo: giantswarm/observability-bundle
     version: 0.8.7
+  teleportKubeAgent:
```

Comment:
@tuladhar Does this require teleport binary in the nodes?

Reply:
No, teleport binary is not required.